main/modes/mode_flight.c

```diff
@@ -1223,9 +1223,9 @@ static void flightGameUpdate( flight_t * tflight )
         tflight->hpr[0] += tflight->pitchmoment;
         tflight->hpr[1] += tflight->yawmoment;
         
-        if( tflight->hpr[0] > 3960 ) tflight->hpr[0] -= 3960;
+        if( tflight->hpr[0] >= 3960 ) tflight->hpr[0] -= 3960;
```

Comment:
This makes a difference? `sin1024[]` is accessed through `getSin1024()`, which should mod the value down to the proper range anyway.
```C
int16_t getSin1024(int16_t degree)
{
    degree = ( (degree % 360) + 360 ) % 360;
    return sin1024[degree];
}
```